Dangerfile

```diff
@@ -19,4 +19,5 @@ has_modified_model = git.modified_files.include? "WordPress/Classes/WordPress.xc
 warn("Core Data: Do not edit an existing model in a release branch unless it hasn't been released to testers yet. Instead create a new model version and merge back to develop soon.") if has_modified_model
 
 # Podfile: no references to commit hashes
-warn("Podfile: reference to a commit hash") if `grep -e "^[^#]*:commit" Podfile`.length > 1
+### (except for Gutenberg)
+warn("Podfile: reference to a commit hash") if `grep -e "^[^#]*:commit" Podfile | grep -v Gutenberg`.length > 1
```

Comment:
This works but `.length > 1` here refers to the character count, not the number of matches which confused me initially. Here is a pure Ruby version that might be a little clearer:

```
warn("Podfile: reference to a commit hash") if File.readlines('Podfile').select{ |l| l[/^[^#]*:commit/] && !l.include?("Gutenberg") }
```

Either way is fine though 😃

Reply:
yep! `grep` output should be an empty string if everything is fine in the file.